### PR TITLE
Blog progress structure

### PR DESCRIPTION
**Pull request by MiyamotoAkira**

### CODE REVIEW

File: src/_posts/2019-06-12-progress-structure.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+image: 
+    src: /assets/custom/img/blog/2019/pattern-split.jpg
+    attribution:
+       text: Photo by Dmitri Popov on Unsplash
+       href: https://unsplash.com/@dmpop
+tags: 
+- codurance
+---
+
+This is the third and last post (at least for now), on internal workings of Codurance. This time, I will be talking about something that we are currently implementing. It is a direct consequence of having the Initiative Circles described on my previous post.
```
Comment (Mattsi-Jansky) [inline src/_posts/2019-06-12-progress-structure.md]: "on internal workings of" -> "on the internal workings of"

Comment (Mattsi-Jansky) [inline src/_posts/2019-06-12-progress-structure.md]: "described on my previous post" -> "described in my previous post"

File: src/_posts/2019-06-12-progress-structure.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+Sandro and Mash (our founders) created Codurance with the idea of having a flat hierarchy. Their own experience lead to this idea. In a company of consultants, with highly skilled employees, a formalized bureaucracy will get in the way of moving forward. Furthermore, in my experience, bureaucracy hinders the agility of a company (but there is always a minimum needed).
```
Comment (Mattsi-Jansky) [inline src/_posts/2019-06-12-progress-structure.md]: "Their own experience lead" -> "Their own experience led"

Comment (Mattsi-Jansky) [inline src/_posts/2019-06-12-progress-structure.md]: "formalized" -> "formalised" (US/UK spelling)

File: src/_posts/2019-06-12-progress-structure.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+A flat hierarchy doesn’t mean that there is no structure to your company. You need structure at two very important points: on dealings with the client, as the structure helps on communication channels; and on the career advancement of your consultants. I am not going to talk about the first type here, though we have been adding that as well. But I will be talking … well, writing ... about the second.
```
Comment (Mattsi-Jansky) [inline src/_posts/2019-06-12-progress-structure.md]: The first ellipsis here is a unicode ellipsis (`…`) and the second is three periods (`...`). It should be consistent, and I'd favour the unicode ver over three periods.

File: src/_posts/2019-06-12-progress-structure.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+At Codurance there was some basic structure: Junior, Craftsperson, Principal, but that structure was barely defined. We kind of knew how a person moved from Junior to Craftsperson, but we did not know how you could move from Craftsperson—and this is a problem. To understand more, I recommend that you watch this [presentation](https://youtu.be/jA1Q94d2z10?list=PLBzScQzZ83I_qiY6iuS-jHmp1QvdE8m5_) by [Marco Rodgers](https://www.linkedin.com/in/marcorogers/) or look at his [slides](https://speakerdeck.com/polotek/creating-a-career-ladder-for-engineers) or do both. But the gist of it is that, without a clear idea of how to move, people get lost in the middle. It was an issue for our juniors because they were left to their own devices to know how to improve, and for senior people because they were left without a way to show their advancement, nor a way to reach a higher position.
```
Comment (Mattsi-Jansky) [inline src/_posts/2019-06-12-progress-structure.md]: Perhaps "to know how to improve" -> "without guidance on how to improve"?

File: src/_posts/2019-06-12-progress-structure.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+
+So out of these concerns we tried earlier to create some kind of structure, but the effort fizzled out. Yet the concern was important enough that we tried a second time, giving ourselves tight deadlines to make sure that we set it up. So a group of people got together and we created a progression structure.
+
+One of the things that we decided at the beginning is that we were not trying to create the perfect structure. We wanted to create something that was good enough, and we will keep evaluating the system and improving it once we started to see the issues with it.
```
Comment (Mattsi-Jansky) [inline src/_posts/2019-06-12-progress-structure.md]: "once we started to see the issues with it." -> "if we see issues with it."

File: src/_posts/2019-06-12-progress-structure.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+
+The other important aspect is that we are a consultancy, and therefore the skills chosen for advancement are to improve our capacity as consultants.
+
+## Progress Structure.
+So how is the progress structure setup? We have a collection of disciplines—areas where we want people to improve. Each discipline is divided into at most 5 levels. Each level has one or more skills/experiences. The idea is that they represent a continuous advancement on the difficulty or knowledge required. Then there are the spider nets, an approximate shape of the levels on each discipline that we expect a person at each level to have reached. We want to allow for some flexibility as some of the “requirements” really depend on being provided with opportunities to develop in these areas. Sometimes people are so well advanced around a specific area that compensates for lack in others. But in general, for a consultancy, well rounded individuals, that can cover all the job/projects possibilities is better.
```
Comment (Mattsi-Jansky) [inline src/_posts/2019-06-12-progress-structure.md]: "disciplines—areas" -> "disciplines— areas" (missing space)

File: src/_posts/2019-06-12-progress-structure.md [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+
+A detail that I believe is important is that your career advancement will not depend on someone of higher seniority level vacating a position. Each person progress is its own, and will not be affected by others progression.
```
Comment (Mattsi-Jansky) [inline src/_posts/2019-06-12-progress-structure.md]: "Each person progress is its own" -> "Each person's progress is their own"

Comment (Mattsi-Jansky) [inline src/_posts/2019-06-12-progress-structure.md]: "and will not be affected by others progression." -> missing posessive apostrophe on "others", but I'd recommend restructuring to something like "and will not be affected by the progression of their peers".

File: src/_posts/2019-06-12-progress-structure.md [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+
+Originally I saw this as trying to get well rounded people that is useful on every possible scenario. But probably the broken comb metaphor is better suited on the long term. The less people you have in your company, the more important it is that anyone can do any job. But as you grow, that importance subsides.
```
Comment (Mattsi-Jansky) [inline src/_posts/2019-06-12-progress-structure.md]: "is useful on" -> "are useful in"

File: src/_posts/2019-06-12-progress-structure.md [hunk 9]
```diff
@@ -0,0 +1,52 @@
...
+
+Finally, the competency matrix that we are using is not a tick-all-boxes idea. It provides a general idea of  expectations.
```
Comment (Mattsi-Jansky) [inline src/_posts/2019-06-12-progress-structure.md]: "Finally, the competency matrix that we are using..." - perhaps flesh this out a bit more. You drop the name "competency matrix" for the first time here and it hasn't been introduced yet. eg "We use a competency matrix, a sort of tick-box exercise that..."

File: src/_posts/2019-06-12-progress-structure.md [hunk 10]
```diff
@@ -0,0 +1,52 @@
...
+
+##  It is a Marathon, not a sprint.
+I think this is something that needs to be understood. It takes years to reach Principal (the highest seniority) level. You need a broad knowledge and experience. You need to have seen multiple projects, with different characteristics, so for any new assignments you can reach into your own drawer of solutions, approaches, tricks to move forward satisfactorily a project. Trying to race here only leads to disappointment. Or as they saying goes: Rome wasn’t built in a day.
+
+## Mentorship
+Mentoring is one of the most important ways in which we can help people to improve and progress. Therefore, we are introducing mentoring (non-compulsory) for every level of seniority. A mentor will become a guide and helper, to allow the mentee to move forwards. It is important however that the mentee the one in charge of his/her own progression. The mentor is there to lend a hand, provide advice, and help with getting Codurance to provide the resource/opportunities/solutions that the mentee required.
```
Comment (Mattsi-Jansky) [inline src/_posts/2019-06-12-progress-structure.md]: "the mentee the one in" -> "the mentee is the one in"

Comment (Mattsi-Jansky) [inline src/_posts/2019-06-12-progress-structure.md]: "that the mentee required." -> "that the mentee requires."

File: src/_posts/2019-06-12-progress-structure.md [hunk 11]
```diff
@@ -0,0 +1,52 @@
...
+
+## Roll Out plan
+
+The roll out plan is all about getting everyone more or less aligned in terms of the future progression, and on the more appropriate seniority level. We do expect people to be all over the place right now, due to the fact that before each was left to find their own path. Furthermore, we have a competency matrix that is live, but doesn’t mean it is perfect. Not only do we need to train ourselves, but the model itself. So a 6 month period has been reserved to get everyone rolling with a plan of improvement, and get additional feedback in the process. An initial assignment of mentors (based on the mentees preferences) has been made, just to make it easy for everyone to start.
```
Comment (Mattsi-Jansky) [inline src/_posts/2019-06-12-progress-structure.md]: "before each was" -> "before everyone was"

Comment (Mattsi-Jansky) [reply src/_posts/2019-06-12-progress-structure.md]: "is live, but doesn’t mean it is" -> "is live, but that doesn’t mean it is"

**Review by Mattsi-Jansky — COMMENTED** [review]
Looks really good, left some grammar/spelling suggestions

File: src/_posts/2019-06-12-progress-structure.md (hunk 7 above)
Comment (MiyamotoAkira) [reply src/_posts/2019-06-12-progress-structure.md]: Added colleagues, rather than peers.

File: src/_posts/2019-06-12-progress-structure.md (hunk 1 above)
Comment (MiyamotoAkira) [reply src/_posts/2019-06-12-progress-structure.md]: done

File: src/_posts/2019-06-12-progress-structure.md (hunk 2 above)
Comment (MiyamotoAkira) [reply src/_posts/2019-06-12-progress-structure.md]: done

File: src/_posts/2019-06-12-progress-structure.md (hunk 1 above)
Comment (MiyamotoAkira) [reply src/_posts/2019-06-12-progress-structure.md]: done

File: src/_posts/2019-06-12-progress-structure.md (hunk 2 above)
Comment (MiyamotoAkira) [reply src/_posts/2019-06-12-progress-structure.md]: done

File: src/_posts/2019-06-12-progress-structure.md (hunk 3 above)
Comment (MiyamotoAkira) [reply src/_posts/2019-06-12-progress-structure.md]: done

File: src/_posts/2019-06-12-progress-structure.md (hunk 8 above)
Comment (MiyamotoAkira) [reply src/_posts/2019-06-12-progress-structure.md]: done

File: src/_posts/2019-06-12-progress-structure.md (hunk 9 above)
Comment (MiyamotoAkira) [reply src/_posts/2019-06-12-progress-structure.md]: Rewritten

File: src/_posts/2019-06-12-progress-structure.md (hunk 10 above)
Comment (MiyamotoAkira) [reply src/_posts/2019-06-12-progress-structure.md]: done

Comment (MiyamotoAkira) [reply src/_posts/2019-06-12-progress-structure.md]: done

File: src/_posts/2019-06-12-progress-structure.md (hunk 11 above)
Comment (MiyamotoAkira) [reply src/_posts/2019-06-12-progress-structure.md]: done

File: src/_posts/2019-06-12-progress-structure.md (hunk 4 above)
Comment (MiyamotoAkira) [reply src/_posts/2019-06-12-progress-structure.md]: done

File: src/_posts/2019-06-12-progress-structure.md (hunk 5 above)
Comment (MiyamotoAkira) [reply src/_posts/2019-06-12-progress-structure.md]: done

File: src/_posts/2019-06-12-progress-structure.md (hunk 7 above)
Comment (MiyamotoAkira) [reply src/_posts/2019-06-12-progress-structure.md]: done

File: src/_posts/2019-06-12-progress-structure.md (hunk 6 above)
Comment (MiyamotoAkira) [reply src/_posts/2019-06-12-progress-structure.md]: done

**Pull request merged**